packages/kotti-ui/source/kotti-button/KtButton.vue

```diff
@@ -69,11 +69,30 @@ export default defineComponent<KottiButton.PropsInternal>({
 					'KtButton: Guideline Uncompliance: attempted to use helpText with a button that already has a label.',
 				)
 			}
+
+			if (
+				props.isToggle &&
```

Comment:
if you remove isToggle from props then here you would check

props.toggleStatus !== null

Reply:
you can even have a computed called isToggle 

so isToggle = computed(() => (props.toggleStatus !== null))